action.yml

```diff
@@ -50,7 +50,7 @@ inputs:
   date-format:
     description: 'Provide a format to use for outputting dates, based on options available in the dateformat npm package.'
     required: false
-    default: false
+    default: 'dd-MM-yyyy'
```

Comment:
Are you sure the default shouldn't be MM-dd-yyyy 😉

Reply:
👿